scripts/zones/Norg/Zone.lua

```diff
@@ -7,11 +7,13 @@ local ID = require("scripts/zones/Norg/IDs")
 require("scripts/globals/conquest")
 require("scripts/globals/keyitems")
 require("scripts/globals/missions")
+require("scripts/globals/quests")
```

Comment:
unused require; not needed.

note that charVars can be accessed without quests require.  quests require is needed for references such as tpz.quest.id.foo.FOO or QUEST_AVAILABLE.